src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1311,4 +1311,12 @@
   <data name="OperatorOverloadsHaveNamedAlternatesCodeFixTitle" xml:space="preserve">
     <value>Add operator overload named alternate</value>
   </data>
-</root>
+  <data name="AvoidMultipleEnumerationsMessage" xml:space="preserve">
+    <value>Whenever an 'IEnumerable' collection is used, it will be enumerated. Try convert it to other collections.</value>
+    <comment>'IEnumerable' is a well-known interface name. It should not be translated</comment>
+  </data>
+  <data name="AvoidMultipleEnumerationsTitle" xml:space="preserve">
+    <value>Possible multiple enumerations over 'IEnumerable' collection</value>
+    <comment>'IEnumerable' is a well-known interface name. It should not be translated</comment>
```

Comment:
There is a special syntax for locking localization.

https://github.com/dotnet/roslyn-analyzers/blob/01f57af818ca7b93e1c27b69e7ad6cfb3d275f82/src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticsAnalyzersResources.resx#L380